### ISSUE TITLE
Contribute to the main Apache project

### ISSUE THREAD

**Issue by guiwoda** [nealio82/avro-php#4]
Hey! Kudos for making this. It looks way better than the official classes.

Have you tried sending this version back their way? I'm sure they'll accept it if you explain the better practices and standards of your code.

Cheers

**Comment by nealio82** [comment]
Hey @guiwoda, thanks!

For one part you've just reminded me about this package - looking at the code there are still improvements to be made (stylistic really, such as adding braces around if conditions which were left out of the official package's code). I'll go through and make adjustment, then send it to Apache for comment.

I'm assuming you found this because you need to use Avro with PHP? If so, I hope it helps you solve a problem!

**Comment by guiwoda** [comment]
Yep, I'm building a small application that will consume and produce events to / from kafka with Avro involved. The PHP ecosystem doesn't have a lot of options for these things!

Cheers!
